static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -455,10 +455,10 @@ require.def(
                     this._emitEvent(MediaPlayer.EVENT.STATUS);
                 }
             },
-            
+
             _updateRange: function () {
                 var self = this;
-                if (this._currentPlayer === this.PlayerEmps.StreamingPlayer) {
+                if (this._isHlsMimeType() && this._isLiveMedia()) {
```

Comment:
`isLiveMedia` is in the superclass https://github.com/bbc/tal/blob/master/static/script/devices/mediaplayer/mediaplayer.js#L320

`isHlsMimeType` is in this file https://github.com/bbc/tal/blob/master/static/script/devices/mediaplayer/samsung_streaming.js#L683